cmd/openshift-install/agent/waitfor.go

```diff
@@ -76,7 +76,7 @@ func newWaitForInstallCompleteCmd() *cobra.Command {
 				cluster.PrintInfraEnvRestAPIEventList()
 				err2 := cluster.API.OpenShift.LogClusterOperatorConditions()
 				if err2 != nil {
-					logrus.Error("Attempted to gather ClusterOperator status after wait failure: ", err2)
+					logrus.Errorf("Attempted to gather ClusterOperator status after wait failure: %s. Agent-based installer succeeded with the cluster installation but the error is caused due to Cluster Operator's status availability conditions.", err2)
```

Comment:
We've tried to keep a behavior as much as possible similar to the default `wait-for` command, so I'd not recommend to have a different error message for [the same cases](https://github.com/openshift/installer/blob/c45c58edee95a632ad3eedf55c2d0e7afa563f5e/cmd/openshift-install/waitfor.go#L51) (ie, in case let's change them both, as the problem would be the same also in the non-agent command)